.pylint.rc

```diff
@@ -0,0 +1,58 @@
+[MASTER]
+jobs=4
+persistent=yes
+suggestion-mode=yes
+unsafe-load-any-extension=no
+
+# Blacklist files or directories (basenames, not paths)
+ignore=
```

Comment:
these are here just as a nice to see for the future, if we need, could be removed though